### ISSUE TITLE
[BUG] anvi-setup-scg-databases --scgs-taxonomy-data-dir <PATH> doesn't pass <PATH> to diamond

### ISSUE THREAD

**Issue by jthiels** [merenlab/anvio#1534]
## Short description of the problem

Using anvi-setup-scg-databases --scgs-taxonomy-data-dir <PATH> doesn't work Installs to idiosyncratic /tmp file.  It's not setting the -d parameter in diamond

## anvi'o version

Replace this text with the output of this command:

```
Anvi'o version ...............................: esther (v6.2)
Profile DB version ...........................: 31
Contigs DB version ...........................: 14
Pan DB version ...............................: 13
Genome data storage version ..................: 6
Auxiliary data storage version ...............: 2
Structure DB version .........................: 1
```

## System info

Please tell us which operating system you are using, and how did you install anvi'o.
Ubuntu 18.04--I installed it with conda and put diamond in the PATH at /usr/local/sbin
## Detailed description of the issue
Have a container that won't be writeable by the user internally as per policy; tried to use --scgs-taxonomy-data-dir to set the directory where the database should be written as /tmp is specific to the node used on the cluster so a more permanent location (/home or another volume mounted on the cluster).  I tried setting the path on the build node to /tmp/testanvio to test but it wrote to an idiosyncratic /tmp file.

**Comment by sklasek** [comment]
I think I'm experiencing a similar problem. I've tried running anvi-setup-scg-taxonomy (master version) and anvi-setup-scg-databases (v. 6.2). Both give me this output:
```
WARNING
===============================================
Anvi'o found your FASTA files in place, but not the databases. Now it will
generate all the search databases using the existing FASTA files.

[30 Oct 20 13:53:16 Creating search databases] Working on Ribosomal_S2 in 8 threads                                                                                  

Config Error: Something went wrong and DIAMOND did not create the database file it was
              supposed to for Ribosomal_S2 :(  
```

**Comment by meren** [comment]
Yes, unfortunately the solution (for now) is to ask the system administrator to run the `anvi-setup-scg-databases` since the `--scgs-taxonomy-data-dir` is not well connected to the module :(

**Comment by jthiels** [comment]
Hi Scott,

My user was able to do what he needed with a combination of the minimal
install into a conda environment someone referenced right after my question
and the singularity container I made.  However, it did create the file in
/tmp so not a long term solution for HPC users if they need a fixed
location for it.

Best,

John

John Thiels
LSA Technology Services
Research Computing and Infrastructure
University of Michigan, Ann Arbor
ARC-TS Support : arcts-support@umich.edu

**Comment by meren** [comment]
Thank you very much for your follow up, @jthiels.

I have to admit that I find it absolutely astonishing to see someone who is responsible for an entire Research Computing and Infrastructure service at a university to take the time to reach out to the developers of an open-source software just because one of their users struggled, and THEN take the time again to actually share their resolution to help other users of the same software.

I wanted to send you my thanks for your generosity.

**Comment by sklasek** [comment]
Hi John,
Late follow-up, but thanks for the info. I got my sysadmin to run this successfully. 
Scott
